tests/Unit/Subscriber/Behavior/SubscriberTestCase.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager\Tests\Unit\Subscriber\Behavior;
+
+class SubscriberTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
I would say YAGNI applies here. As long as we don't put any code in here I would not add another layer of inheritance. Also, we have a class like this in sulu/sulu, and I don't really like it... It's instantiating a few classes, which are not needed by all sub classes. I would prefer to also get rid of it there.

Summed up, I would not add empty classes to the inheritance hierarchy, because we might end up using it one time. If we do that all the time we would have a very bloated inheritance hierarchy.